service/co/hotwax/product/ProductFacilityServices.xml

```diff
@@ -133,9 +133,16 @@
                 <return message="No products falls under any rule in this rule group."/>
             </if>
 
+            <!--Writing directly to DB instead of putting it on SFTP and then other job reading and processing it from SFTP-->
+            <script>
+                recordsLoaded = ec.entity.makeDataLoader().location(csvFilePath).csvEntityName("org.apache.ofbiz.product.facility.ProductFacility").load()
```

Comment:
This will rollback complete transaction if single records fails, also it's not good idea to use load method for large data set. This will use single transaction to commit all the records .